main.py

```
@@ -2,47 +2,52 @@
 from agents.executor import execute_task
-from agents.mutator import mutate_protocol, analyze_all_protocols
+from agents.mutator import mutate_protocol
 from utils.logger import log
 from utils.tracker import get_protocol_stats
 import sys
 
-def run_self_correcting_executor(protocol='default_protocol', iterations=1):
+
+def run_self_correcting_executor(protocol="default_protocol", iterations=1):
     """Run the self-correcting executor with automatic mutation"""
     log(f"🚀 Starting Self-Correcting MCP Executor")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)